en/docs/guides/login/configure-ws-federation-single-sign-on.md

```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
+
+4. Click **List** under **Users and Roles** and edit Alex's user profile. Enter an email address for Alex.
+
+
+
+
+
+### Download the sample
+
+To be able to deploy a WSO2 Identity Server sample, you need to download
+it onto your machine first.
+
+Follow the instructions below to download a sample from GitHub.
+
+1. Navigate to [WSO2 Identity Server Samples](https://github.com/wso2/samples-is/releases).
+2. [Download](https://github.com/wso2/samples-is/releases/download/v4.3.0/PassiveSTSSampleApp.war) the `PassiveSTSSampleApp.war` file from the
+   latest release assets.
+
+### Deploying the PassiveSTSSampleApp webapp
+
+Deploy this sample web app on a web container.
+
+1.  Copy the .war file into the `           webapps          `
+    folder. For example,
+    `           <TOMCAT_HOME>/apache-tomcat-<version>/webapps          `
+    .
+2.  Start the Tomcat server.
+    
+
+### Configure sample properties
+
+Open the `<TOMCAT_HOME>/apache-tomcat-<version>/webapps/PassiveSTSSampleApp/WEB-INF/web.xml` file and configure the following. 
+    
+1.  Specify ` idpUrl ` as the URL of Identity Server's Passive STS. The
+    following is an example.
+
+    ``` xml
+    <init-param>
+            <param-name>idpUrl</param-name>
+            <param-value>https://localhost:9443/passivests</param-value>
+    </init-param> 
+    ```
+
+2.  Specify the ` replyURL ` as the URL of the web app. The following
+    specifies ` PassiveSTSSampleApp ` as the web app.
+
+    ``` xml
+    <init-param>
+            <param-name>replyUrl</param-name>
+            <param-value>http://localhost:8080/PassiveSTSSampleApp/index.jsp</param-value>
+    </init-param>
+    ```
+
+3.  Specify the ` realm ` . This should be a unique identifier for the
+    web app as seen in the example below.
+
+    ``` xml
+    <init-param>
+            <param-name>realm</param-name>
+            <param-value>PassiveSTSSampleApp</param-value>
+    </init-param> 
+    ```
+    
+4.  For tenant user logins, specify the ` tenantDomain ` .
+
+    ``` xml
+    <init-param>
+            <param-name>requestParams</param-name>
+            <param-value>tenantDomain=tenant4.com</param-value>
+    </init-param>
+    ```
+    
+5. Restart the tomcat server.
+
+To check the sample application,
+navigate to `http://<TOMCAT_HOST>:<TOMCAT_PORT>/PassiveSTSSampleApp/home.jsp ` on your browser.
+
+For example, ` http://localhost:8080/PassiveSTSSampleApp/home.jsp .`
+
+
+### Configuring the service provider
+
+The next step is to configure the service provider.
+
+1.  Return to the WSO2 IS management console.
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **Add**.
+
+3.  Enter **PassiveSTSSampleApp** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4.  In the **Inbound Authentication Configuration** section, set following configurations
+    under the **WS-Federation (Passive) Configuration** section.
+
+    1.  
```

Comment:
should this be removed?

Reply:
commit 13cc337adbc3c694220614bbf71b3e343379056b